app/views/shared/_invitations.haml

```diff
@@ -1,15 +1,9 @@
-= t('.share_this')
+= t(".share_this")
 = invite_link(current_user.invitation_code)
-#invitations-button.btn.btn-link{ "data-toggle" => "modal"}
+#invitations-button.invitations-link.btn.btn-link{"data-toggle" => "modal"}
```

Comment:
Classes should be listed before IDs (.invitations-link should precede #invitations-button)